dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -235,6 +244,23 @@ export const Card = ({
 		);
 	};
 
+	if (type === 'LinkSnap' && enriched?.embedHtml !== undefined) {
+		return (
+			<div css={snapStyles}>
+				<Global styles={enriched?.embedCss} />
```

Comment:
Can we / should we try to namespace this CSS so that it can't affect other page elements?